content/en/about-xlt/000-demo/index.md

```diff
@@ -101,13 +104,24 @@ Another image using the `imageres` shortcode, which will resize images to a smal
 ## Colored Text
 Should you need colored text, use {{< ctext color="green" >}}ctext in green{{< /ctext >}} or any other html compatible color code. If none is given, this defaults to {{< ctext >}}grey{{< /ctext >}}.
 
-## //TODO
-{{< TODO >}}
-To remind us that something needs to be done, it is colored very annoyingly. Like, get those images resized by Docsy instead of providing two image files. 
+## TODO
+### To-Do Marker
+{{< TODO / >}}To remind us that something needs to be done, it introduces a marked TODO at the position of the shortcode `{{</* TODO / */>}}`. TODO markers right now assume to be in the beginning because they make a little room on the right side - "{{< TODO / >}}".
 
-Update: TODOs are not displayed any more (content is not rendered by Docsy), but are still in markdown code to mark the places where something needs to be done. Tasks should also be added to Github once this shortcode is used.
-{{< /TODO >}}
+{{< TODO comment="I am more useful!" / >}}Optionally you can pass the parameter comment and provide some more information such as `{{</* TODO comment="More information in the title" / */>}}`.
 
+### Marked Text
+```
+{{</* TODO */>}}To mark some text use this.{{</* /TODO */>}}
+```
+{{< TODO >}}To mark some text use this.{{< /TODO >}}
+
+
+### Markdown in a To-Do
```

Comment:
This is the case for all shortcodes, so I guess whether you use {{% or {{< is more of a docsy thing? Not necessarily part of this short doc page

Reply:
It makes a difference. <> results in text being printed, while %% results in markdown being processed inside the tag. Let's to different results at the end aka with and without formatting. I see that as doc for the doc writers to avoid having that someplace else, at least for the moment.